ProjetSupaero2018/catkin_ws/src/display/scripts/XY_Emulator.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
```

Comment:
I would say, do not commit this one. It may be kept but does not appear to be necessary